src/app/components/client/ExposureCard.tsx

```diff
@@ -132,48 +134,74 @@ export const ExposureCard = (props: Props) => {
+              {
+                elems: {"data_broker_link": <a href={props.exposureDetailsLink}/>}
+              })}
+              <a href={exposureDetailsLink}>
+                <span>
+                  <OpenInNew alt="" width="13" height="13" />
+                </span>
+              </a>
+              {". "}{l10n.getString("exposure-card-description-info-for-sale-part-two")}
+            </p>
+          </div> : 
+          // Data breach content
+          <div>
+            <p>
+              {l10n.getFragment("exposure-card-description-data-breach-part-one", 
+              {
+                vars: {
+                  data_breach_company: props.exposureName,
+                  data_breach_date: formattedDate
+                },
+                elems: {"data_breach_link": <a href={props.exposureDetailsLink}/>}
+              })}
+              <a href={exposureDetailsLink}>
+                <span>
+                  <OpenInNew alt="" width="13" height="13" />
+                </span>
+              </a>
+              {". "}{l10n.getString("exposure-card-description-data-breach-part-two")}
+            </p>
+          </div>
+          }
           <div className={styles.exposureListOfExposureTypes}>
-            <ul>
-              <li>Your exposed info:</li>
-              <li>
+            <dl>
+              <div>{l10n.getString("exposure-card-your-exposed-info")}:</div>
+              <div>
                 <DetailsFoundItem
-                  icon={<MultipleUsers alt="" width="15" height="15" />}
+                  icon={<MultipleUsers alt="" width="13" height="13" />}
                   whichExposed="family"
                   num={0}
                 />
-              </li>
-              <li>
+              </div>
+              <div>
                 <DetailsFoundItem
-                  icon={<PhoneIcon alt="" width="15" height="15" />}
+                  icon={<PhoneIcon alt="" width="13" height="13" />}
                   whichExposed="phone"
                   num={5}
                 />
-              </li>
-              <li>
+              </div>
+              <div>
                 <DetailsFoundItem
-                  icon={<EmailIcon alt="" width="15" height="15" />}
+                  icon={<EmailIcon alt="" width="13" height="13" />}
                   whichExposed="email"
                   num={4}
                 />
-              </li>
-              <li>
+              </div>
+              <div>
                 <DetailsFoundItem
-                  icon={<LocationPin alt="" width="15" height="15" />}
+                  icon={<LocationPin alt="" width="13" height="13" />}
                   whichExposed="address"
                   num={0}
                 />
-              </li>
-            </ul>
+              </div>
+            </dl>
             <span className={styles.fixItBtn}>
               <Button type={"primary"} content={"Lets fix it"} />
```

Comment:
nit: this should be localized

Reply:
The "Lets fix it" looks hardcoded though, and Ctrl+Fing doesn't seem to show any results in any .ftl files (at least not on this page).

```sh
git grep -in "Lets fix it" | cat

src/app/components/client/ExposureCard.tsx:186:              <Button type={"primary"} content={"Lets fix it"} />
```